resources/eventing/charts/controller/templates/certmanager.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.enableNewCRDVersion }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: eventing-serving-cert
+  namespace: kyma-system
+spec:
+  dnsNames:
+    - eventing-controller-webhook-service.kyma-system.svc
+    - eventing-controller-webhook-service.kyma-system.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: eventing-selfsigned-issuer
+  secretName: eventing-webhook-server-cert
```

Comment:
maybe let's define this name in the helpers or values, so we can reuse it in the deployment then ?

Reply:
Sure, keep it like this.